apps/cloud/static/invite.html

```diff
@@ -266,5 +399,10 @@
         </footer>
       </div>
     </div>
+    <script type="text/javascript">
+      var date = new Date();
+      var year = date.getFullYear();
+      document.getElementById("year").innerHTML = year;
```

Comment:
> This is a [security risk](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations) if the string to be inserted might contain potentially malicious content.

`Element.innerHTML` should be used carefully, even though there is no problem using it in this scenario, I would prefer [Node.textContent](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) when inserting plain text, as this inserts it as raw text rather than parsing it as HTML.